test/functional/rpc_net.py

```diff
@@ -319,7 +319,7 @@ def test_getnodeaddresses(self):
 
     def test_addpeeraddress(self):
         self.log.info("Test addpeeraddress")
-        self.restart_node(1, ["-checkaddrman=1", "-test=addrman"])
+        self.restart_node(1, ["-checkaddrman=1", "-test=addrman"], clear_addrman=True)
```

Comment:
Sounds good, I'll add you comment. I haven't seen a CI failure in `rpc_net.py` yet, but just from looking at the code it seemed that this could fail too.